tests/internal/e2e/test/specs/vcast_build_env_failure.test.ts

```diff
@@ -0,0 +1,242 @@
+// Test/specs/vcast.test.ts
```

Comment:
Has prettier diffs

Reply:
Executed prettier in 11217bc161def3093067a7a263374993e8ab8a30.